conf/flight_plans/tudelft/flight_plan.dtd

```diff
@@ -0,0 +1,341 @@
+<!-- Paparazzi flight plan DTD -->
```

Comment:
Better to use the `flight_plan.dtd` that is in `conf/flight_plans` than to copy the file over.